vowpalwabbit/contcb.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "reductions.h"
+
+namespace CONTCB
```

Comment:
>CONTCB [](start = 10, length = 6)

VW format for namespaces is getting standardized.  

I would suggest the following:
namespace VW { namespace continuous_cb { }